src/styles/getModalStyles.js

```diff
@@ -20,10 +23,9 @@ export default (type, windowDimensions) => {
             // and can be dismissed by clicking outside of the modal.
             // This modal should take up the entire visible area when
             // viewed on a smaller device (e.g. mobile or mobile web).
-            modalStyle = {
-                margin: 0,
+            modalStyle = _.extend(modalStyle, {
```

Comment:
NAB, I _believe_ we have more or less standardized on using spread operator instead of `_.extend` but could be wrong about that.

Reply:
Actually yes double checked, there are 0 usages in the codebase. For consistency, maybe we can stick to just using spread operator? Mostly my concern is that we will have 2 ways of doing one thing. Just like we shouldn't be mixing `Array.map` and `_.map` we shouldn't be using `{...something}` and `_.extend(something, {});`